owners/index.js

```diff
@@ -14,17 +14,19 @@
  * limitations under the License.
  */
 
-const sleep = require('sleep-promise');
+const Octokit = require('@octokit/rest');
 const {GitHub, PullRequest} = require('./src/github');
 const {LocalRepository} = require('./src/local_repo');
-const {CheckRun, OwnersCheck} = require('./src/owners_check');
+const {OwnersBot} = require('./src/owners_bot');
+const {OwnersParser} = require('./src/owners');
+const {OwnersCheck} = require('./src/owners_check');
 
-const GITHUB_CHECKRUN_DELAY = 2000;
+const GITHUB_TOKEN = process.env.GITHUB_TOKEN;
 
 module.exports = app => {
-  app.on(['pull_request.opened', 'pull_request.synchronize'], onPullRequest);
-  app.on('check_run.rerequested', onCheckRunRerequest);
-  app.on('pull_request_review.submitted', onPullRequestReview);
+  const localRepo = new LocalRepository(process.env.GITHUB_REPO_DIR);
+  const ownersBot = new OwnersBot(localRepo);
+  const router = app.route('/admin');
```

Comment:
Calling this `router` is kinda like creating a C++ variable with `int number = 0;` - technically correct, but there's gotta be a better name 😝
Maybe `adminRouter`?
(edit: totally aware that I probably did the same thing in one of the other apps 🙃)

Reply:
Done.

Reply:
Done.